napari_allencell_segmenter/view/batch_processing_view.py

```diff
@@ -89,15 +87,6 @@ def update_button(self, enabled: bool):
         """
         self.btn_run_batch.setEnabled(enabled)
 
-    def open_completion_dialog(self, output_folder: Path):
-        """
-        Open the batch processing completion dialog box
-        Inputs:
-            output_folder (Path): output folder to open when prompted by user
-        """
-        dlg = BatchCompleteDialog(output_folder)
-        dlg.exec()
```

Comment:
I'm not too sure about moving this code since it should live in the view, but since we use this method once I think it should be fine. Maybe add a comment about what the two lines do in the controller

Reply:
Yeah, agreed it's not optimal but that was my rationale, it's only once..... Will add comment about what it does and also a TODO about moving it back to the view at some point